packages/web/src/js/Tooltip.ts

```diff
@@ -12,16 +13,57 @@ const EVENT_HIDDEN = `hidden${EVENT_KEY}`;
 const EVENT_SHOW = `show${EVENT_KEY}`;
 const EVENT_SHOWN = `shown${EVENT_KEY}`;
 
+const ARROW_SELECTOR = '[data-spirit-element="arrow"]';
 const CLASS_NAME_VISIBLE = 'is-visible';
 const CLASS_NAME_HIDDEN = 'is-hidden';
 
+type Config = {
+  flip: boolean;
+  shift: boolean;
+  size: boolean;
+  flipCrossAxis: boolean;
+  flipFallbackAxisSideDirection: 'none' | 'start' | 'end';
+  flipFallbackPlacements: FloatingUI.Placement[];
+  placement: FloatingUI.Placement;
+  placementControlled: boolean;
+};
+
+export const transformStringToArray = (str: FloatingUI.Placement[]) => {
```

Comment:
I think I do not understand this function correctly. The input type is an Array and the output is the same type. The name of the function suggests, that you are transforming a string to an array but the function itself is doing something else. Please, choose a better name.

Reply:
function and types fixed